icu4c/source/i18n/unicode/unounclass.h

```diff
@@ -0,0 +1,29 @@
+// © 2022 and later: Unicode, Inc. and others.
+// License & terms of use: http://www.unicode.org/copyright.html
+
+#ifndef __UNOUNCLASS_H__
+#define __UNOUNCLASS_H__
+
+#include "unicode/uversion.h"
+
+U_NAMESPACE_BEGIN
+
+/**
+ * Represents all the grammatical class nouns that are supported by CLDR.
```

Comment:
Please add `@draft ICU 71` and wrap this with the ifdef guard, and make sure `hdrtst` is happy (it runs on CI)